CITATION.cff

```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+title: bokeh-a11y-audi
+message: >-
+  If you use this resource, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: "Frank"
+    family-names: "Elavsky"
+    orcid: "https://orcid.org/0000-0002-6849-5893"
+    affiliation: "Carnegie Mellon University"
+  - given-names: "Gabriel"
+    family-names: "Fouasnon"
+    affiliation: "Quansight Labs"
+  - given-names: "Mateusz"
+    family-names: "Paprocki"
+    affiliation: "Bokeh, Anaconda"
+  - given-names: "Pavithra"
+    family-names: "Eswaramoorthy"
+    affiliation: "Bokeh, Quansight Labs"
+  - given-names: "Tania"
+    family-names: "Allard"
+    affiliation: "Quansight Labs"
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.14923641
+    description: Repository DOI
+repository-code: >-
+  https://github.com/Quansight-Labs/bokeh-a11y-audit/tree/main
+url: "https://bokeh-a11y-audit.readthedocs.io/"
+abstract: >-
+  Findings and progress of the Bokeh accessibility
+  audit as part of the Accessible interactive data
+  visualizations in Python with
+  Bokeh grant funded by the
+  CZI EOSS Cycle 6.
```

Comment:
@frankelavsky would you be able to suggest a better abstract or some improvements? brain is not braining 🧠

Reply:
Honestly, this is really good and succinct. Is there a word/character limit at all? If we wanted a slightly more descriptive abstract, we could frame it/motivate it with a couple sentences. Something like, "Interactive data science tools currently exclude people with disabilities from participating in their capabilities. In this work, we conducted an accessibility audit of Bokeh, an interactive visualization library for Python, to identify evidence of barriers and areas for improvement. This document presents our findings and progress of that audit as part of the ..."

Reply:
AFAIK there is no limit. Will update with your suggestion.